zebra/zebra_vty.c

```diff
@@ -120,6 +120,109 @@ DEFUN (ip_multicast_mode,
 	return CMD_SUCCESS;
 }
 
+DEFUN(show_jack_sparrow, show_jack_sparrow_cmd, "show jack sparrow",
```

Comment:
hmm - is this like a test to see if the reviewers actually look at the diffs?

Reply:
yeah I was just fucking around in this PR for someone else.  I'll get rid of this eventually

Reply:
removesd